diff --git a/node_modules/material-ui-search-bar/lib/components/SearchBar/SearchBar.js b/node_modules/material-ui-search-bar/lib/components/SearchBar/SearchBar.js
index bc4e497..ee6de2b 100644
--- a/node_modules/material-ui-search-bar/lib/components/SearchBar/SearchBar.js
+++ b/node_modules/material-ui-search-bar/lib/components/SearchBar/SearchBar.js
@@ -9,17 +9,17 @@ var _react = _interopRequireDefault(require("react"));
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
-var _IconButton = _interopRequireDefault(require("@material-ui/core/IconButton"));
+var _IconButton = _interopRequireDefault(require("@mui/material/IconButton"));
 
-var _Input = _interopRequireDefault(require("@material-ui/core/Input"));
+var _Input = _interopRequireDefault(require("@mui/material/Input"));
 
-var _Paper = _interopRequireDefault(require("@material-ui/core/Paper"));
+var _Paper = _interopRequireDefault(require("@mui/material/Paper"));
 
-var _Clear = _interopRequireDefault(require("@material-ui/icons/Clear"));
+var _Clear = _interopRequireDefault(require("@mui/icons-material/Clear"));
 
-var _Search = _interopRequireDefault(require("@material-ui/icons/Search"));
+var _Search = _interopRequireDefault(require("@mui/icons-material/Search"));
 
-var _withStyles = _interopRequireDefault(require("@material-ui/core/styles/withStyles"));
+var _withStyles = _interopRequireDefault(require("@mui/styles/withStyles"));
 
 var _classnames = _interopRequireDefault(require("classnames"));
 
